AnVIL snapshots with null in anvil_file.file_size column

… which may cause IT failures in `hammerbox`.

https://gitlab.explore.anvilproject.org/ucsc/azul/-/jobs/48340
```
ERROR: test_indexing (integration_test.IndexingIntegrationTest.test_indexing) [repository_files] (catalog='anvil10-it')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builds/ucsc/azul/test/integration_test.py", line 366, in subTest
    yield
  File "/builds/ucsc/azul/test/integration_test.py", line 1162, in _test_repository_files
    self._validate_file_response(response, source, inner_file)
  File "/builds/ucsc/azul/test/integration_test.py", line 1197, in _validate_file_response
    self._validate_file_content(response, file)
  File "/builds/ucsc/azul/test/integration_test.py", line 1180, in _validate_file_content
    self.assertEqual(1 if file['size'] > 0 else 0, len(content.read(1)))
                          ^^^^^^^^^^^^^^^^
TypeError: '>' not supported between instances of 'NoneType' and 'int'
```

Datasets with files containing `files.file_size` set to `null`:
```
❯ http 'https://service.explore.anvilproject.org/index/datasets?catalog=anvil9&filters=%7B%0A%20%20%22files.file_size%22%3A%20%7B%22is%22%3A%20%5Bnull%5D%7D%0A%7D&size=10' | jq -r '.hits[].sources[].source_spec'
tdr:bigquery:gcp:datarepo-d4765cad:AnVIL_ENCORE_RS293_20250304_ANV5_202503042020:/0
tdr:bigquery:gcp:datarepo-9ebf9c60:AnVIL_ENCORE_293T_20250304_ANV5_202503042023:/0
```

After we added stronger type checking in the metadata plugin (partial PR #7352 for #6821), our workarounds aren't sufficient anymore because the defective file entities are now checked on the server side where we can't work around them that easily. The resulting 500 error causes the IT to fail

https://gitlab.explore.anvilproject.org/ucsc/azul/-/jobs/54714

The server logs:

**CloudWatch Logs Insights**    
region: us-east-1    
log-group-names: /aws/lambda/azul-service-hammerbox    
start-time: -3600s    
end-time: 0s    
query-string:
  ```
  fields @message
| filter @message like '3447a277-8a00-47d6-98d4-61692cced94e'
| sort @timestamp asc
| limit 10000
  ```
---
| @message |
| --- |
| START RequestId: 3447a277-8a00-47d6-98d4-61692cced94e Version: $LATEST |
| [INFO] 2025-10-18T05:29:58.301Z 3447a277-8a00-47d6-98d4-61692cced94e azul.chalice Received GET request for &#x27;&#x2f;fetch&#x2f;repository&#x2f;files&#x2f;02e9c8c4-f197-4a61-8eec-89a7c541f8ae&#x27;, with {&quot;query&quot;: {&quot;catalog&quot;: &quot;anvil11-it&quot;, &quot;version&quot;: &quot;2022-06-01T00:00:00.000000Z&quot;}, &quot;headers&quot;: {&quot;accept-encoding&quot;: &quot;identity&quot;, &quot;cloudfront-forwarded-proto&quot;: &quot;https&quot;, &quot;cloudfront-is-desktop-viewer&quot;: &quot;true&quot;, &quot;cloudfront-is-mobile-viewer&quot;: &quot;false&quot;, &quot;cloudfront-is-smarttv-viewer&quot;: &quot;false&quot;, &quot;cloudfront-is-tablet-viewer&quot;: &quot;false&quot;, &quot;cloudfront-viewer-asn&quot;: &quot;14618&quot;, &quot;cloudfront-viewer-country&quot;: &quot;US&quot;, &quot;host&quot;: &quot;service.hammerbox.explore.anvilproject.org&quot;, &quot;user-agent&quot;: &quot;python-urllib3&#x2f;1.26.20&quot;, &quot;via&quot;: &quot;1.1 ef81d2c0d5984a166a5467acd7c2d88a.cloudfront.net (CloudFront)&quot;, &quot;x-amz-cf-id&quot;: &quot;cKbC2MoRz1usvTX4CrUiEpOU_MCNigtUedJK0-HRxN-Ye0ZUvWSOeg==&quot;, &quot;x-amzn-trace-id&quot;: &quot;Root=1-68f325d6-56949e151d42f9d14a69667a&quot;, &quot;x-forwarded-for&quot;: &quot;35.168.152.160, 15.158.60.239&quot;, &quot;x-forwarded-port&quot;: &quot;443&quot;, &quot;x-forwarded-proto&quot;: &quot;https&quot;}}. |
| [INFO] 2025-10-18T05:29:58.302Z 3447a277-8a00-47d6-98d4-61692cced94e azul.chalice … without a request body |
| [INFO] 2025-10-18T05:29:58.302Z 3447a277-8a00-47d6-98d4-61692cced94e azul.chalice Did not authenticate request. |
| [INFO] 2025-10-18T05:29:58.306Z 3447a277-8a00-47d6-98d4-61692cced94e azul.boto3 dynamodb.GetItem: Making POST request to https:&#x2f;&#x2f;dynamodb.us-east-1.amazonaws.com&#x2f; |
| [INFO] 2025-10-18T05:29:58.306Z 3447a277-8a00-47d6-98d4-61692cced94e azul.boto3 dynamodb.GetItem: … with a request body of length 142 being b&#x27;{&quot;TableName&quot;: &quot;azul-sources_cache_by_auth-hammerbox&quot;, &quot;Key&quot;: {&quot;identity&quot;: {&quot;S&quot;: &quot;anvil11-it:&quot;}}, &quot;ProjectionExpression&quot;: &quot;sources,expiration&quot;}&#x27; |
| [INFO] 2025-10-18T05:29:58.310Z 3447a277-8a00-47d6-98d4-61692cced94e azul.boto3 dynamodb.GetItem: Got 200 response |
| [INFO] 2025-10-18T05:29:58.310Z 3447a277-8a00-47d6-98d4-61692cced94e azul.boto3 dynamodb.GetItem: … with a response body of length 70159 starting in b&#x27;{&quot;Item&quot;:{&quot;expiration&quot;:{&quot;N&quot;:&quot;1760765380&quot;},&quot;sources&quot;:{&quot;S&quot;:&quot;[\\&quot;440838b4-2c24-4961-8079-89f5041c90b7\\&quot;, \\&quot;e55c8632-3b03-49c8-8452-d386486e7f9e\\&quot;, \\&quot;a29fcfcc-86a2-4f56-bdd7-43b75958af29\\&quot;, \\&quot;be095362-152e-48ac-a825-2dc5022698c6\\&quot;, \\&quot;69f9a48a-ca5e-4a51-ae60-fc91ac20352a\\&quot;, \\&quot;9971cf89-1eb2-40cd-8b38-679e45d64b2e\\&quot;, \\&quot;1a8e001e-13a9-4f4c-b4e2-3ff3152d0027\\&quot;, \\&quot;7a8a3267-5d84-408b-80f3-456f1081f680\\&quot;, \\&quot;03e7d636-469b-4311-a5fa-a17799776b8a\\&quot;, \\&quot;6abbe02d-b89d-49b2-b639-b45f21d5baa7\\&quot;, \\&quot;8f24d669-3567-4596-81e0-15de1c42f608\\&quot;, \\&quot;05e95b36-cbd0-4ed9-9ed9-117e72726c25\\&quot;, \\&quot;1ae5a119-fbac-4ec1-a3b5-c6a54abf22cf\\&quot;, \\&quot;125f9819-bdd5-4606-88dd-07e312d3059d\\&quot;, \\&quot;1413a6bf-c209-4a69-90d1-82390611fff1\\&quot;, \\&quot;2bb7a279-d3d0-4e1b-b1eb-3cf0ba721b0c\\&quot;, \\&quot;744d9c13-2a15-4ecc-8c65-a2e349d04336\\&quot;, \\&quot;161757fc-d932-4d12-aa24-74f3309e4775\\&quot;, \\&quot;fd1bdf73-98e0-410c-939f-2800bcdd001f\\&quot;, \\&quot;9bd7ba07-3434-417c-803c-82f48e040aac\\&quot;, \\&quot;954a9b7e-403c-43a9-8d6b-639e6f37bdd9\\&quot;, \\&quot;2d2bf1f0-0b6d-4557-b142-9804f2d0b155\\&quot;, \\&quot;990dfe4e-bbfe-4926-85f8-ebbd608dc2c3\\...&#x27; |
| [INFO] 2025-10-18T05:29:58.311Z 3447a277-8a00-47d6-98d4-61692cced94e azul.terra Making GET request to &#x27;https:&#x2f;&#x2f;data.terra.bio&#x2f;api&#x2f;repository&#x2f;v1&#x2f;snapshots&#x2f;roleMap&#x27; |
| [INFO] 2025-10-18T05:29:58.311Z 3447a277-8a00-47d6-98d4-61692cced94e azul.terra … without a request body |
| [INFO] 2025-10-18T05:29:58.311Z 3447a277-8a00-47d6-98d4-61692cced94e azul.terra … with request headers [(&#x27;Connection&#x27;, &#x27;close&#x27;), (&#x27;x-goog-api-client&#x27;, &#x27;cred-type&#x2f;sa&#x27;), (&#x27;authorization&#x27;, &#x27;REDACTED&#x27;)] |
| [INFO] 2025-10-18T05:29:58.754Z 3447a277-8a00-47d6-98d4-61692cced94e azul.terra Got 200 response after 0.443s from GET to https:&#x2f;&#x2f;data.terra.bio&#x2f;api&#x2f;repository&#x2f;v1&#x2f;snapshots&#x2f;roleMap |
| [INFO] 2025-10-18T05:29:58.754Z 3447a277-8a00-47d6-98d4-61692cced94e azul.terra … with response headers HTTPHeaderDict({&#x27;Date&#x27;: &#x27;Sat, 18 Oct 2025 05:29:58 GMT&#x27;, &#x27;Server&#x27;: &#x27;Apache&#x27;, &#x27;X-Frame-Options&#x27;: &#x27;SAMEORIGIN&#x27;, &#x27;Access-Control-Allow-Headers&#x27;: &#x27;DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range,Authorization,Accept,Referer,X-App-Id,Origin&#x27;, &#x27;Access-Control-Allow-Methods&#x27;: &#x27;GET,POST,DELETE,PUT,PATCH,OPTIONS,HEAD&#x27;, &#x27;X-Content-Type-Options&#x27;: &#x27;nosniff&#x27;, &#x27;Strict-Transport-Security&#x27;: &#x27;max-age=31536000;includeSubDomains&#x27;, &#x27;Cache-Control&#x27;: &#x27;no-cache,no-store,must-revalidate&#x27;, &#x27;X-Request-ID&#x27;: &#x27;a00Wjzj1&#x27;, &#x27;Content-Type&#x27;: &#x27;application&#x2f;json&#x27;, &#x27;Content-Length&#x27;: &#x27;145228&#x27;, &#x27;Set-Cookie&#x27;: &#x27;JSESSIONID=1B620D0ABF3B3DAB39AF876C6BC0FC37; Path=&#x2f;; Secure; HttpOnly&#x27;, &#x27;Vary&#x27;: &#x27;Accept-Encoding,Origin&#x27;, &#x27;Via&#x27;: &#x27;1.1 google&#x27;, &#x27;Alt-Svc&#x27;: &#x27;h3=&quot;:443&quot;; ma=2592000,h3-29=&quot;:443&quot;; ma=2592000&#x27;, &#x27;Connection&#x27;: &#x27;close&#x27;}) |
| [INFO] 2025-10-18T05:29:58.754Z 3447a277-8a00-47d6-98d4-61692cced94e azul.terra … with a response body of length 145228 starting in b&#x27;{&quot;roleMap&quot;:{&quot;81e4a4c1-2341-46f6-9af1-005097500086&quot;:[&quot;reader&quot;,&quot;discoverer&quot;,&quot;aggregate_data_reader&quot;],&quot;c5aeee20-d383-4c09-8a78-a7c810af7857&quot;:[&quot;reader&quot;,&quot;discoverer&quot;,&quot;aggregate_data_reader&quot;],&quot;82336861-0e39-497f-bdbe-03923f5f36ab&quot;:[&quot;reader&quot;,&quot;discoverer&quot;,&quot;aggregate_data_reader&quot;],&quot;4a94d70f-5335-410c-a0b0-909226481259&quot;:[&quot;reader&quot;,&quot;discoverer&quot;,&quot;aggregate_data_reader&quot;],&quot;bc7f344d-ff18-4a10-8f59-518324ad4126&quot;:[&quot;reader&quot;,&quot;discoverer&quot;,&quot;aggregate_data_reader&quot;],&quot;c25f206f-c5de-48f2-9dfa-7ab41221d6eb&quot;:[&quot;reader&quot;,&quot;discoverer&quot;,&quot;aggregate_data_reader&quot;],&quot;bbe3aa2e-42b0-44db-83c8-a4b9ec8c31ab&quot;:[&quot;reader&quot;,&quot;discoverer&quot;,&quot;aggregate_data_reader&quot;],&quot;96bd5e38-1f50-48d8-a78b-00d89a1ad105&quot;:[&quot;reader&quot;,&quot;discoverer&quot;,&quot;aggregate_data_reader&quot;],&quot;0cadeadc-7470-4e57-8de8-a3e320c3e6a8&quot;:[&quot;reader&quot;,&quot;discoverer&quot;,&quot;aggregate_data_reader&quot;],&quot;5397053b-33a7-456d-9b2c-66c65483734e&quot;:[&quot;reader&quot;,&quot;discoverer&quot;,&quot;aggregate_data_reader&quot;],&quot;a17642d0-5b55-407d-b205-e7a849a49bc7&quot;:[&quot;reader&quot;,&quot;discoverer&quot;,&quot;aggregate_data_reader&quot;],&quot;a0299a62-22f3-497c-bacb-53f1a2d9b387&quot;:[&quot;reader&quot;,&quot;di...&#x27; |
| [INFO] 2025-10-18T05:29:58.759Z 3447a277-8a00-47d6-98d4-61692cced94e azul.boto3 dynamodb.PutItem: Making POST request to https:&#x2f;&#x2f;dynamodb.us-east-1.amazonaws.com&#x2f; |
| [INFO] 2025-10-18T05:29:58.760Z 3447a277-8a00-47d6-98d4-61692cced94e azul.boto3 dynamodb.PutItem: … with a request body of length 70252 starting in b&#x27;{&quot;TableName&quot;: &quot;azul-sources_cache_by_auth-hammerbox&quot;, &quot;Item&quot;: {&quot;identity&quot;: {&quot;S&quot;: &quot;anvil11-it:&quot;}, &quot;sources&quot;: {&quot;S&quot;: &quot;[\\&quot;440838b4-2c24-4961-8079-89f5041c90b7\\&quot;, \\&quot;e55c8632-3b03-49c8-8452-d386486e7f9e\\&quot;, \\&quot;a29fcfcc-86a2-4f56-bdd7-43b75958af29\\&quot;, \\&quot;be095362-152e-48ac-a825-2dc5022698c6\\&quot;, \\&quot;69f9a48a-ca5e-4a51-ae60-fc91ac20352a\\&quot;, \\&quot;9971cf89-1eb2-40cd-8b38-679e45d64b2e\\&quot;, \\&quot;1a8e001e-13a9-4f4c-b4e2-3ff3152d0027\\&quot;, \\&quot;7a8a3267-5d84-408b-80f3-456f1081f680\\&quot;, \\&quot;03e7d636-469b-4311-a5fa-a17799776b8a\\&quot;, \\&quot;6abbe02d-b89d-49b2-b639-b45f21d5baa7\\&quot;, \\&quot;8f24d669-3567-4596-81e0-15de1c42f608\\&quot;, \\&quot;05e95b36-cbd0-4ed9-9ed9-117e72726c25\\&quot;, \\&quot;1ae5a119-fbac-4ec1-a3b5-c6a54abf22cf\\&quot;, \\&quot;125f9819-bdd5-4606-88dd-07e312d3059d\\&quot;, \\&quot;1413a6bf-c209-4a69-90d1-82390611fff1\\&quot;, \\&quot;2bb7a279-d3d0-4e1b-b1eb-3cf0ba721b0c\\&quot;, \\&quot;744d9c13-2a15-4ecc-8c65-a2e349d04336\\&quot;, \\&quot;161757fc-d932-4d12-aa24-74f3309e4775\\&quot;, \\&quot;fd1bdf73-98e0-410c-939f-2800bcdd001f\\&quot;, \\&quot;9bd7ba07-3434-417c-803c-82f48e040aac\\&quot;, \\&quot;954a9b7e-403c-43a9-8d6b-639e6f37bdd9\\&quot;, \\&quot;2d2bf1f0-0b6d-4557-b1...&#x27; |
| [INFO] 2025-10-18T05:29:58.772Z 3447a277-8a00-47d6-98d4-61692cced94e azul.boto3 dynamodb.PutItem: Got 200 response |
| [INFO] 2025-10-18T05:29:58.772Z 3447a277-8a00-47d6-98d4-61692cced94e azul.boto3 dynamodb.PutItem: … with a response body of length 2 being b&#x27;{}&#x27; |
| [INFO] 2025-10-18T05:29:58.776Z 3447a277-8a00-47d6-98d4-61692cced94e opensearch Making POST request to https:&#x2f;&#x2f;vpc-azul-index-anvilprod-ggipah4skn2ftt47u4xgvydzqm.us-east-1.es.amazonaws.com:443&#x2f;azul_v2_hammerbox_anvil11-it_files_aggregate&#x2f;_search |
| [INFO] 2025-10-18T05:29:58.776Z 3447a277-8a00-47d6-98d4-61692cced94e opensearch … with a request body of length 65405 starting in b&#x27;{&quot;query&quot;:{&quot;bool&quot;:{&quot;must&quot;:[{&quot;constant_score&quot;:{&quot;filter&quot;:{&quot;terms&quot;:{&quot;contents.files.uuid.keyword&quot;:[&quot;02e9c8c4-f197-4a61-8eec-89a7c541f8ae&quot;]}}}},{&quot;constant_score&quot;:{&quot;filter&quot;:{&quot;terms&quot;:{&quot;contents.files.version.keyword&quot;:[&quot;2022-06-01T00:00:00.000000Z&quot;]}}}},{&quot;constant_score&quot;:{&quot;filter&quot;:{&quot;terms&quot;:{&quot;sources.id.keyword&quot;:[&quot;004ee0a4-7672-4922-9ec2-9f1c4e1ff4a9&quot;,&quot;0070f6a4-5430-47aa-b1fb-f7d175a470c8&quot;,&quot;009e77b1-362e-47e0-9ae7-a03a390a07ce&quot;,&quot;00c1a1f1-6bfc-478b-8b32-5d5911081638&quot;,&quot;00d1422e-2055-4a74-af2e-c7ab1e25cd5e&quot;,&quot;01795dd6-5123-4ed4-88de-06a1a99f75af&quot;,&quot;01a8a72d-0a21-4bbf-b50f-6f1687cf1e61&quot;,&quot;01d0e707-2021-4740-9b21-60c8ad24fc76&quot;,&quot;01e26559-f5a5-42f8-b601-c24ef42f4aba&quot;,&quot;01ebc9c6-08de-41e5-a689-aaa5685a1d92&quot;,&quot;020e1f1a-7314-44b5-a281-942a45269815&quot;,&quot;02137a6c-e68a-4ccb-b5aa-e1013d8e44da&quot;,&quot;023486a4-2a09-4a19-aba4-f717a7483e6e&quot;,&quot;02378204-6234-413a-8905-95cf2454de93&quot;,&quot;026ea946-27df-403e-8215-511f1cdf2537&quot;,&quot;029f160a-b25f-44cd-ad7e-98ee3709135b&quot;,&quot;02a58bdd-4afb-4d36-ae6d-ebe8b1f3f951&quot;,&quot;02bb75f3-e9c1-4ae4-bf99-ec8115cb1adf&quot;,&quot;02cb6c0a-504...&#x27; |
| [INFO] 2025-10-18T05:29:58.788Z 3447a277-8a00-47d6-98d4-61692cced94e opensearch Got 200 response after 0.012s from POST to https:&#x2f;&#x2f;vpc-azul-index-anvilprod-ggipah4skn2ftt47u4xgvydzqm.us-east-1.es.amazonaws.com:443&#x2f;azul_v2_hammerbox_anvil11-it_files_aggregate&#x2f;_search |
| [INFO] 2025-10-18T05:29:58.788Z 3447a277-8a00-47d6-98d4-61692cced94e opensearch … with a response body of length 3330 starting in &#x27;{&quot;took&quot;:4,&quot;timed_out&quot;:false,&quot;_shards&quot;:{&quot;total&quot;:1,&quot;successful&quot;:1,&quot;skipped&quot;:0,&quot;failed&quot;:0},&quot;hits&quot;:{&quot;total&quot;:{&quot;value&quot;:1,&quot;relation&quot;:&quot;eq&quot;},&quot;max_score&quot;:3.0,&quot;hits&quot;:[{&quot;_index&quot;:&quot;azul_v2_hammerbox_anvil11-it_files_aggregate&quot;,&quot;_id&quot;:&quot;02e9c8c4-f197-4a61-8eec-89a7c541f8ae&quot;,&quot;_score&quot;:3.0,&quot;_source&quot;:{&quot;entity_id&quot;:&quot;02e9c8c4-f197-4a61-8eec-89a7c541f8ae&quot;,&quot;contents&quot;:{&quot;activities&quot;:[{&quot;document_id&quot;:[&quot;512b4c65-e281-4af0-b651-23eb6926e83d&quot;,&quot;5b5c4c8e-9338-49a5-91e7-20f95e1cc2e4&quot;,&quot;a1ff2f15-0172-43b4-8c4a-3d806a315570&quot;,&quot;e0c651f8-3a6e-40c9-8fa8-8989129b965e&quot;,&quot;fe097c52-e5b2-4dc3-9fac-7847b9c168a8&quot;],&quot;source_datarepo_row_ids&quot;:[&quot;activity:07d53719-bedc-4ce3-a1da-5855a9e37ac3&quot;,&quot;activity:0a3cb753-c05b-4a19-bfb8-20592458b991&quot;,&quot;activity:1480927c-1f6b-434e-ac46-7dbbd9b41d80&quot;,&quot;activity:6ec7ead1-f502-4e95-9dfb-fad4b65be63f&quot;,&quot;activity:86f5a484-bf6e-4089-83f3-ee903d191911&quot;],&quot;activity_id&quot;:[&quot;20951de9-0478-d86c-8a1c-d15ca92b9a49&quot;,&quot;b863debd-2625-d6af-9691-da97ae589e1b&quot;,&quot;cf2e1312-ba77-9e5b-cee4-c7fcf3af6637&quot;,&quot;e0adfee5-7c65-6302-0bf9-708eee52a1ce&quot;,&quot;fc229270-28c…&#x27; |
| [ERROR] 2025-10-18T05:29:58.789Z 3447a277-8a00-47d6-98d4-61692cced94e azul-service-hammerbox Caught exception for path &#x2f;fetch&#x2f;repository&#x2f;files&#x2f;{file_uuid} Traceback (most recent call last):   File &quot;&#x2f;var&#x2f;task&#x2f;chalice&#x2f;app.py&quot;, line 1922, in _get_view_function_response     response = view_function(**function_args)   File &quot;&#x2f;var&#x2f;task&#x2f;app.py&quot;, line 1568, in fetch_repository_files     body = _repository_files(file_uuid, fetch=True)   File &quot;&#x2f;var&#x2f;task&#x2f;app.py&quot;, line 1582, in _repository_files     return app.repository_controller.download_file(catalog=catalog,            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^                                                    fetch=fetch,                                                    ^^^^^^^^^^^^     ...&lt;2 lines&gt;...                                                    headers=headers,                                                    ^^^^^^^^^^^^^^^^                                                    authentication=request.authentication)                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File &quot;&#x2f;var&#x2f;task&#x2f;azul&#x2f;service&#x2f;repository_controller.py&quot;, line 236, in download_file     file = self.service.get_data_file(catalog=catalog,                                       file_uuid=file_uuid,                                       file_version=file_version,                                       filters=self.get_filters(catalog, authentication, None))   File &quot;&#x2f;var&#x2f;task&#x2f;azul&#x2f;service&#x2f;repository_service.py&quot;, line 347, in get_data_file     file = plugin.file_class.from_index(file)   File &quot;&#x2f;var&#x2f;task&#x2f;azul&#x2f;plugins&#x2f;metadata&#x2f;anvil&#x2f;__init__.py&quot;, line 547, in from_index     size=json_int(hit[&#x27;file_size&#x27;]),          ~~~~~~~~^^^^^^^^^^^^^^^^^^   File &quot;&#x2f;var&#x2f;task&#x2f;azul&#x2f;types.py&quot;, line 170, in json_int     assert isinstance(v, int), type(v)            ~~~~~~~~~~^^^^^^^^ AssertionError: &lt;class &#x27;NoneType&#x27;&gt; |
| [INFO] 2025-10-18T05:29:58.792Z 3447a277-8a00-47d6-98d4-61692cced94e azul.chalice Returning 500 response with headers {&quot;headers&quot;: {&quot;Access-Control-Allow-Origin&quot;: &quot;*&quot;, &quot;Access-Control-Allow-Headers&quot;: &quot;Authorization,Content-Type,X-Amz-Date,X-Amz-Security-Token,X-Api-Key&quot;, &quot;Content-Security-Policy&quot;: &quot;default-src &#x27;self&#x27;;img-src &#x27;self&#x27; data:;script-src &#x27;self&#x27;;style-src &#x27;self&#x27;;frame-ancestors &#x27;none&#x27;;form-action &#x27;self&#x27;&quot;, &quot;Referrer-Policy&quot;: &quot;strict-origin-when-cross-origin&quot;, &quot;Strict-Transport-Security&quot;: &quot;max-age=63072000; includeSubDomains; preload&quot;, &quot;X-Content-Type-Options&quot;: &quot;nosniff&quot;, &quot;X-Frame-Options&quot;: &quot;DENY&quot;, &quot;X-XSS-Protection&quot;: &quot;1; mode=block&quot;, &quot;Cache-Control&quot;: &quot;no-store&quot;}}. |
| [INFO] 2025-10-18T05:29:58.792Z 3447a277-8a00-47d6-98d4-61692cced94e azul.chalice … with a response body of length 133 being {&quot;Code&quot;: &quot;InternalServerError&quot;, &quot;Message&quot;: &quot;An internal server error occurred.&quot;, &quot;RequestId&quot;: &quot;3447a277-8a00-47d6-98d4-61692cced94e&quot;} |
| END RequestId: 3447a277-8a00-47d6-98d4-61692cced94e |
| REPORT RequestId: 3447a277-8a00-47d6-98d4-61692cced94e Duration: 492.47 ms Billed Duration: 493 ms Memory Size: 2048 MB Max Memory Used: 176 MB |
---

According to 

https://service.explore.anvilproject.org/index/files?catalog=anvil11&filters=%7B%0A%20%20%22files.file_size%22%3A%20%7B%0A%20%20%20%20%22is%22%3A%20%5B%0A%20%20%20%20%20%20null%0A%20%20%20%20%5D%0A%20%20%7D%0A%7D&size=10 

the affected datasets are AnVIL_ENCORE_RS293 and AnVIL_ENCORE_293T, which were recently updated with new snapshots  (#7465).

--- a/deployments/anvilprod/environment.py
+++ b/deployments/anvilprod/environment.py
@@ -1195,7 +1195,7 @@ anvil11_sources = mkdict(anvil10_sources, 367, mkdelta([
     bqsrc('61b6b42b', 'ccdg_broad_np_epilepsy_zafagn_ds_epi_como_mds_wes_20221026_ANV5_202409302116', pop),
 ]))
 
-anvil12_sources = mkdict(anvil11_sources, 369, mkdelta([
+anvil12_sources = mkdict(anvil11_sources, 372, mkdelta([
     bqsrc('48204a7f', 'CCDG_Broad_CVD_EOCAD_PartnersBiobank_HMB_WES_20230621_ANV5_202510141553'),
     bqsrc('50aed057', 'CCDG_Broad_NP_Epilepsy_AUSAUS_EP_BA_CN_ID_MDS_GSA_MD_20251016_ANV5_202510161935'),
     bqsrc('25e5ec27', 'CCDG_Broad_NP_Epilepsy_USAMON_GRU_WES_20250721_ANV5_202510141558'),
@@ -1220,14 +1220,18 @@ anvil12_sources = mkdict(anvil11_sources, 369, mkdelta([
     bqsrc('5447de30', 'CMH_GAFK_WGBS_20230327_ANV5_202402062120', pop),
     bqsrc('de339830', 'CMH_GAFK_WGS_20240113_ANV5_202502201948', pop),
     bqsrc('ba97c05c', 'CMH_GAFK_scRNA_20221107_ANV5_202402291004', pop),
+    bqsrc('76ceb22a', 'ENCORE_293T_20251112_ANV5_202511121451'),
+    bqsrc('3b16bd9c', 'ENCORE_RS293_20251112_ANV5_202511121422'),
     bqsrc('4096189d', 'GREGOR_R04_GRU_20250917_ANV5_202509221559'),
     bqsrc('f71485a8', 'GREGOR_R04_HMB_20250910_ANV5_202509221540'),
     bqsrc('3b09455f', 'GREGoR_R01_GRU_20240208_ANV5_202510101245'),
     bqsrc('25d953fe', 'GTEx_public_data_20240117_ANV5_202510101251'),
     bqsrc('193dc871', 'GTEx_v10_hg38_20241105_ANV5_202510141951'),
+    bqsrc('3dddda4d', 'HPRC_20240401_ANV5_202511121431'),
     bqsrc('33f117db', 'IGVF_GRU_PUB_R1_20251015_ANV5_202510151322'),
     bqsrc('ad1d9ffd', 'IGVF_HMB_MDS_R1_20251028_ANV5_202510281333'),
     bqsrc('ca277df1', 'IGVF_HMB_R1_20251015_ANV5_202510151330'),
+    bqsrc('7c901939', 'IGVF_Mouse_R1_20251112_ANV5_202511121500'),
     bqsrc('0570db33', 'MAS_ISO_seq_20240113_ANV5_202510141607'),
     bqsrc('5614f9b2', 'NIMH_Broad_ConvergentNeuro_McCarroll_Eggan_CIRM_GRU_VillageData_20230109_ANV5_202510141609'),
     bqsrc('a314d3c7', 'OurHealth_GRU_R1_20251015_ANV5_202510151901'),

--- a/deployments/anvilprod/environment.py
+++ b/deployments/anvilprod/environment.py
@@ -1195,7 +1195,7 @@ anvil11_sources = mkdict(anvil10_sources, 367, mkdelta([
     bqsrc('61b6b42b', 'ccdg_broad_np_epilepsy_zafagn_ds_epi_como_mds_wes_20221026_ANV5_202409302116', pop),
 ]))
 
-anvil12_sources = mkdict(anvil11_sources, 372, mkdelta([
+anvil12_sources = mkdict(anvil11_sources, 373, mkdelta([
     bqsrc('48204a7f', 'CCDG_Broad_CVD_EOCAD_PartnersBiobank_HMB_WES_20230621_ANV5_202510141553'),
     bqsrc('50aed057', 'CCDG_Broad_NP_Epilepsy_AUSAUS_EP_BA_CN_ID_MDS_GSA_MD_20251016_ANV5_202510161935'),
     bqsrc('25e5ec27', 'CCDG_Broad_NP_Epilepsy_USAMON_GRU_WES_20250721_ANV5_202510141558'),
@@ -1220,6 +1220,7 @@ anvil12_sources = mkdict(anvil11_sources, 372, mkdelta([
     bqsrc('5447de30', 'CMH_GAFK_WGBS_20230327_ANV5_202402062120', pop),
     bqsrc('de339830', 'CMH_GAFK_WGS_20240113_ANV5_202502201948', pop),
     bqsrc('ba97c05c', 'CMH_GAFK_scRNA_20221107_ANV5_202402291004', pop),
+    bqsrc('3e69f21f', 'Complex_Chromosomal_Changes_R1_20251110_ANV5_202511101946'),
     bqsrc('76ceb22a', 'ENCORE_293T_20251112_ANV5_202511121451'),
     bqsrc('3b16bd9c', 'ENCORE_RS293_20251112_ANV5_202511121422'),
     bqsrc('4096189d', 'GREGOR_R04_GRU_20250917_ANV5_202509221559'),

--- a/deployments/hammerbox/environment.py
+++ b/deployments/hammerbox/environment.py
@@ -1197,7 +1197,7 @@ anvil11_sources = mkdict(anvil10_sources, 367, mkdelta([
     bqsrc('61b6b42b', 'ccdg_broad_np_epilepsy_zafagn_ds_epi_como_mds_wes_20221026_ANV5_202409302116', pop),
 ]))
 
-anvil12_sources = mkdict(anvil11_sources, 372, mkdelta([
+anvil12_sources = mkdict(anvil11_sources, 373, mkdelta([
     bqsrc('48204a7f', 'CCDG_Broad_CVD_EOCAD_PartnersBiobank_HMB_WES_20230621_ANV5_202510141553'),
     bqsrc('50aed057', 'CCDG_Broad_NP_Epilepsy_AUSAUS_EP_BA_CN_ID_MDS_GSA_MD_20251016_ANV5_202510161935'),
     bqsrc('25e5ec27', 'CCDG_Broad_NP_Epilepsy_USAMON_GRU_WES_20250721_ANV5_202510141558'),
@@ -1222,6 +1222,7 @@ anvil12_sources = mkdict(anvil11_sources, 372, mkdelta([
     bqsrc('5447de30', 'CMH_GAFK_WGBS_20230327_ANV5_202402062120', pop),
     bqsrc('de339830', 'CMH_GAFK_WGS_20240113_ANV5_202502201948', pop),
     bqsrc('ba97c05c', 'CMH_GAFK_scRNA_20221107_ANV5_202402291004', pop),
+    bqsrc('3e69f21f', 'Complex_Chromosomal_Changes_R1_20251110_ANV5_202511101946'),
     bqsrc('76ceb22a', 'ENCORE_293T_20251112_ANV5_202511121451'),
     bqsrc('3b16bd9c', 'ENCORE_RS293_20251112_ANV5_202511121422'),
     bqsrc('4096189d', 'GREGOR_R04_GRU_20250917_ANV5_202509221559'),

--- a/deployments/hammerbox/environment.py
+++ b/deployments/hammerbox/environment.py
@@ -1197,7 +1197,7 @@ anvil11_sources = mkdict(anvil10_sources, 367, mkdelta([
     bqsrc('61b6b42b', 'ccdg_broad_np_epilepsy_zafagn_ds_epi_como_mds_wes_20221026_ANV5_202409302116', pop),
 ]))
 
-anvil12_sources = mkdict(anvil11_sources, 369, mkdelta([
+anvil12_sources = mkdict(anvil11_sources, 372, mkdelta([
     bqsrc('48204a7f', 'CCDG_Broad_CVD_EOCAD_PartnersBiobank_HMB_WES_20230621_ANV5_202510141553'),
     bqsrc('50aed057', 'CCDG_Broad_NP_Epilepsy_AUSAUS_EP_BA_CN_ID_MDS_GSA_MD_20251016_ANV5_202510161935'),
     bqsrc('25e5ec27', 'CCDG_Broad_NP_Epilepsy_USAMON_GRU_WES_20250721_ANV5_202510141558'),
@@ -1222,14 +1222,18 @@ anvil12_sources = mkdict(anvil11_sources, 369, mkdelta([
     bqsrc('5447de30', 'CMH_GAFK_WGBS_20230327_ANV5_202402062120', pop),
     bqsrc('de339830', 'CMH_GAFK_WGS_20240113_ANV5_202502201948', pop),
     bqsrc('ba97c05c', 'CMH_GAFK_scRNA_20221107_ANV5_202402291004', pop),
+    bqsrc('76ceb22a', 'ENCORE_293T_20251112_ANV5_202511121451'),
+    bqsrc('3b16bd9c', 'ENCORE_RS293_20251112_ANV5_202511121422'),
     bqsrc('4096189d', 'GREGOR_R04_GRU_20250917_ANV5_202509221559'),
     bqsrc('f71485a8', 'GREGOR_R04_HMB_20250910_ANV5_202509221540'),
     bqsrc('3b09455f', 'GREGoR_R01_GRU_20240208_ANV5_202510101245'),
     bqsrc('25d953fe', 'GTEx_public_data_20240117_ANV5_202510101251'),
     bqsrc('193dc871', 'GTEx_v10_hg38_20241105_ANV5_202510141951'),
+    bqsrc('3dddda4d', 'HPRC_20240401_ANV5_202511121431'),
     bqsrc('33f117db', 'IGVF_GRU_PUB_R1_20251015_ANV5_202510151322'),
     bqsrc('ad1d9ffd', 'IGVF_HMB_MDS_R1_20251028_ANV5_202510281333'),
     bqsrc('ca277df1', 'IGVF_HMB_R1_20251015_ANV5_202510151330'),
+    bqsrc('7c901939', 'IGVF_Mouse_R1_20251112_ANV5_202511121500'),
     bqsrc('0570db33', 'MAS_ISO_seq_20240113_ANV5_202510141607'),
     bqsrc('5614f9b2', 'NIMH_Broad_ConvergentNeuro_McCarroll_Eggan_CIRM_GRU_VillageData_20230109_ANV5_202510141609'),
     bqsrc('a314d3c7', 'OurHealth_GRU_R1_20251015_ANV5_202510151901'),

